docs/refguide/api.md

```diff
@@ -520,6 +520,22 @@ Use this if you want to check whether you are using computed properties without
 configure({ computedRequiresReaction: true });
 ```
 
+#### `observablesRequiresReaction: boolean`
+Warns about any unobserved observable access.  
+Use this if you want to check whether you are using observables without a reactive context (eg react component without observer wrapping).
```

Comment:
Reading it now like that, I wonder, are those two options kinda mutually exclusive? I mean when you would use observable in a component without an observer, it would trigger both, wouldn't? That could be more confusing than helpful imo.

Reply:
They kinda complete each other.

example user story:
I developer might have an observer component, but he made the access to the observable value behind an `if` that is not based on observable value (local state or regular prop)
on that case, `derivationRequiresObservable` will help him. (same for autorun/reaction)

Maybe that why its good they will be opt-in

Reply:
I am more worried about other scenario where you forget observer and then you would be getting a warning from both of these checks if I understand it correctly.

We should definitely let the whole thing boil a bit more and try it in React world.

Reply:
If you forgot an observer, you won't get the `derivationRequiresObservable` warn,
because you are not inside a `derivation`